Specs/Overlay Support Spec.md

```diff
@@ -0,0 +1,106 @@
+# Feature Specification: Overlays Support
+
+## Objective
+Support use of Overlays for enabling developers to enhance existing OpenAPI description files without changing the original file.
+
+## Problem Statement
+Existing OpenAPI documents used for AI plugin creation might lack necessary properties or require modifications for them to provide a high quality AI plugin. Direct editing of the original OpenAPI document is often undesirable or impractical.
```

Comment:
Not AI plugin, but I'd even say referenced OpenAPI descriptions that participate in the GPTs and Plugins ecosystem.

Reply:
Yeah, you have phrased it better. I'll rephrase in the spec to capture this.

Reply:
Why restrict the use-case of overlays to AI plug-ins when OpenAPI.NET library caters to a diverse range of users?

Reply:
Sure @adhiambovivian . For copilot scenarios we might would want to provide instructions to the orchestrator on how to use the certain function (/createSong as an exmaple), add localization, provide information on how to present the response to the user (like using adaptive cards or other rendering) etc.